infix-to-postfix.c

```diff
@@ -0,0 +1,114 @@
+#include<stdio.h>
+#include<string.h>
```

Comment:
You may want to put this file in a folder inside Task 3 subdirectory. Please have a look at the `README.md` file. 

Please also add a small description & sample input/output cases in a multiline comment on the top.

Reply:
Please move your file into a folder named `InfixToPostfix`, under `Task 3` folder.